libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
```

Comment:
the timePeriods inventory has been mved to a new constant. in file stok-utils.ts
this is a better way of managing inventory and promote reuse as well.